phpBB/includes/mcp/mcp_logs.php

```diff
@@ -140,7 +140,7 @@ function main($id, $mode)
 						$conditions['topic_id'] = $topic_id;
 					}
 
-					$phpbb_log->delete('mod', $conditions);
+					//$phpbb_log->delete('mod', $conditions);
```

Comment:
It's commented out for future (Should their be conflicts) and surely that's what git blame is for?